pkg/controller/apis/config/types.go

```diff
@@ -84,6 +85,9 @@ type KubeControllerManagerConfiguration struct {
 	// EndpointSliceMirroringControllerConfiguration holds configuration for
 	// EndpointSliceMirroringController related features.
 	EndpointSliceMirroringController endpointslicemirroringconfig.EndpointSliceMirroringControllerConfiguration
+	// EphemeralControllerConfiguration holds configuration for EphemeralController
+	// related features.
+	EphemeralController ephemeralconfig.EphemeralControllerConfiguration
```

Comment:
suggest naming this EphemeralVolumeController for clarity (same for the type)

Reply:
(also applies to the v1alpha1/types.go file)

Reply:
done